.hooks/pre-commit

```diff
@@ -7,7 +7,7 @@ if [ $fmtcount -gt 0 ]; then
 fi
 
 # Due to the way composites work, vet will fail for some of our tests so we ignore it
-vetcount=`go tool vet --composites=false ./ 2>&1  | wc -l`
+vetcount=`go tool vet --composites=false --methods=false ./ 2>&1  | wc -l`
```

Comment:
Ah, I see you also used `SeekTo`, according to a comment below. So can this `go vet` change be pulled?